Core/include/Acts/Seeding2/ITripletSeedCuts.hpp

```diff
@@ -43,9 +43,8 @@ class ITripletSeedCuts {
 
   /// @param seedCandidates contains collection of seed candidates created for one middle
   /// space point in a std::tuple format
-  /// @return vector of seed candidates that pass the cut
   virtual void cutPerMiddleSp(
-      std::span<TripletCandidate2> seedCandidates) const = 0;
+      std::span<TripletCandidate2>& seedCandidates) const = 0;
```

Comment:
Does this make a difference? I imagine having 1 pointer instead of 1 pointer and one size is slightly, but is that all? Aside from that, how does this method work anyway? It does not return, does it modify the span? Is there an example of an implementation anywhere yet?